docs/quickstart/cpp.md

```diff
@@ -1,7 +1,280 @@
 ---
-bodyclass: docs
-headline: C++ Quick Start
 layout: docs
-sidenav: doc-side-quickstart-nav.html
-type: markdown
----
\ No newline at end of file
+title: C++ Quickstart
+---
+
+<h1 class="page-header">C++ Quickstart</h1>
+
+<p class="lead">This guide gets you started with gRPC in C++ with a simple
+working example.</p>
+
+<div id="toc"></div>
+
+## Before you begin
+
+### Prerequisites
+
+#### Install gRPC
+
+To install gRPC on your system, follow the instructions to build from source
+[here](https://github.com/grpc/grpc/blob/master/INSTALL.md).
+
+#### Install Protocol Buffers v3
+
+While not mandatory, gRPC usually leverages Protocol Buffers v3 for service
+definitions and data serialization. If you don't already have it installed on
+your system, you can install the version cloned alongside gRPC:
+
+```sh
+$ git clone -b $(curl -L http://grpc.io/release) https://github.com/grpc/grpc
+$ cd grpc/third_party/protobuf
+$ sudo make install
```

Comment:
which dependencies? for the compilation/installation I'm mostly pointing the reader to grpc/INSTALL.md